src/api/controllers/articleController.js

```diff
@@ -75,6 +76,75 @@ export default class ArticleController {
     });
   }
 
+  /**
+   * This function gets all the articles that are stored in the article table
+   *
+   * @static
+   * @param {Object} req the request
+   * @param {Object} res the response to be sent
+   * @memberof Articles
+   * @returns {Object} res
+   */
+  static async getArticles(req, res) {
+    // default constants
+    const defaultLimit = 10;
+    const defaultOffset = 0;
+
+    const page = req.query.page || 1;
+    const limit = req.query.limit || defaultLimit;
+    let offset = req.query.offset || defaultOffset;
+    const articles = await Article.findAndCountAll();
```

Comment:
@truestbyheart I'm still not sure why you're making two separate queries here. One using `findAndCountAll` and the other using `findAll`.

Also, I think there's a pagination helper @nkpremices created to help with returning the pagination metadata.

Reply:
@truestbyheart please, check how Premices has implemented pagination in the ratings PR. Making two queries here isn't necessary.